robot.js

```diff
@@ -96,16 +103,28 @@
 
     switch (robot.dir) {
       case 'up':
-        y = y < board.length - 1 ? y + 1 : y;
+        // y = y < board.length - 1 ? y + 1 : y;
+        if (y < board.length - 1) {
```

Comment:
You can take out the commented-out code when you submit a pull-request. Consider a pull request to be your final version (and not a draft version) that you want to present to the reviewer. Reviewers are in general not interested in how your code used to be. :grin: